src/azure-cli/azure/cli/command_modules/appconfig/_featuremodels.py

```diff
@@ -94,23 +114,37 @@ class FeatureFlag:
         Description of Feature Flag
     :ivar bool locked:
         Represents whether the feature flag is locked.
+    :ivar str display_name:
+        Display name of the feature flag.
     :ivar str last_modified:
         A str representation of the datetime object representing the last time the feature flag was modified.
     :ivar str etag:
         The ETag contains a value that you can use to perform operations.
     :ivar dict {string, FeatureFilter[]} conditions:
-        Dictionary that contains client_filters List (and server_filters List in future)
-    '''
-
-    def __init__(self,
-                 name,
-                 key=None,
-                 label=None,
-                 state=None,
-                 description=None,
-                 conditions=None,
-                 locked=None,
-                 last_modified=None):
+        Dictionary that contains client_filters List
+    :ivar dict allocation:
+        Determines how variants should be allocated for the feature to various users.
+    :ivar list FeatureVariant variants:
+        The list of variants defined for this feature.
+    :ivar dict {string, bool} telemetry:
```

Comment:
telemetry is dictionary of string and object (enabled property is bool, and metadata property is a dictionary of string key and string value)